pkgs/top-level/stage.nix

```diff
@@ -185,10 +185,25 @@ let
     # that target system. For instance, pkgsCross.raspberryPi.hello,
     # will refer to the "hello" package built for the ARM6-based
     # Raspberry Pi.
+    #
+    # This is essentially the same thing as pkgsLocal except for crossSystem
+    # instead of localSystem.
     pkgsCross = lib.mapAttrs (n: crossSystem:
                               nixpkgsFun { inherit crossSystem; })
                               lib.systems.examples;
 
+    # This maps each entry in lib.systems.examples to its own package set where
+    # packages are built and run on that system.  For example,
+    # pkgsLocal.x86_64-linux.hello will refer to the "hello" package bult for
+    # x86_64-linux.
+    #
+    # This is essentially the same thing as pkgsCross except for localSystem
+    # instead of crossSystem.
+    pkgsLocal =
```

Comment:
"Local" isn't really an appropriate term for cross compilation, unfortunately (and we should get rid of it). It is only appropriate in the context of remote builds, but those are not _handled_ by Nixpkgs. Alternatively, "local" may be applicable in the context of a single derivation, but that's also not our context here.

`pkgsStraight` or `pkgsNative` may be better names.

@Ericson2314 any ideas or suggestions for how to name this package set relation?